src/main/java/io/anserini/index/IndexInfo.java

```diff
@@ -351,7 +351,7 @@ public enum IndexInfo {
           "https://rgw.cs.uwaterloo.ca/pyserini/indexes/lucene/lucene-hnsw-int8.msmarco-v2.1-doc-segmented-shard09.arctic-embed-l.20250114.4884f5.tar.gz" },
       "5e580bb7eb9ee2bb6bfa492b3430c17d",
       IndexType.DENSE_HNSW,
-      "ArcticEmbedLEncoder.classEncoder.class",
+      "ArcticEmbedLEncoder.class",
```

Comment:
here and elsewhere - it'd be a better idea to have the class itself, rather than a String of the class.

not really part of this PR, so you can save for later... or include it here.